bin/check-mesos.rb

```diff
@@ -29,29 +29,26 @@
 require 'sensu-plugin/check/cli'
 require 'rest-client'
 
-# Mesos default ports are defined here: http://mesos.apache.org/documentation/latest/configuration
-MASTER_DEFAULT_PORT = '5050'.freeze
-SLAVE_DEFAULT_PORT = '5051'.freeze
-
 class MesosNodeStatus < Sensu::Plugin::Check::CLI
   option :server,
          description: 'Mesos servers, comma separated',
          short: '-s SERVER1,SERVER2,...',
          long: '--server SERVER1,SERVER2,...',
          default: 'localhost'
 
-  option :mode,
```

Comment:
> This won't be a problem in this case. You can still specify both mode and port and if you only specify the mode the script will assume the default port for that mode but if you specify both it will use what you specified. In this case mode is mandatory but port is optional. This means that for default installations of mesos and DC/OS it will work just with mode. If you use a custom port you'll have to specify it.
Not sure how this is true if you are mreving the option to specify.